compiler/src/dotty/tools/repl/ReplCompiler.scala

```diff
@@ -235,10 +235,14 @@ class ReplCompiler(val directory: AbstractFile) extends Compiler {
                             EmptyValDef,
                             state.imports.map(_._1) :+ valdef)
 
+        val pos =
+          if (trees.isEmpty) Position(0, expr.length)
+          else Position(trees.head.pos.start, trees.last.pos.end)
```

Comment:
@felixmulder Should the position always be `Position(0, expr.length)`?

Reply:
You're creating the wrapper here - so it needs to have a position that spans all the trees, looks equivalent :)

Reply:
Then I would rather not special case